docgram/README.md

```diff
@@ -0,0 +1,185 @@
+Grammar extraction tool for documentation
```

Comment:
I am reticent to have new directories at the root of the tree.

Reply:
Easy to move.  Where do you suggest?  I was mimicking coqpp.

Reply:
I'd say either in `doc/tools/coqrst` or in `coqpp/docgram`. In any case, there are users of Coq that would like to be able to use coqrst to document their libraries and this tool could also be useful to them. We have to think what is the best way to distribute it.

Reply:
How about `doc/tools/docgram`?  doc_grammar is not part of coqrst, which is written in Python.  One directory per tool.

How does `doc/tools` relate to `tools/coqdoc`?  Should they be closer together in the source tree?

Reply:
Sounds good, and yes I think so.